jsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Just wondering on this; should this be something we start configuring in all our repos for VSCode users?

It's a bit annoying VSCode "leaks" this configuration.

Reply:
I agree it’s not ideal but I think their intention is standardising javascript projects through config in a similar manner to the enhancements that `tsconfig` offers. I’d definitely be in favour of adopting in other projects, it’s a good stopgap for providing dev exp benefits without using strict typing in the short term.

Reply:
Let's do it then!